docs/knowledge/ceph/benchmarking.md

```diff
@@ -2,9 +2,182 @@
 title: "Benchmarking"
 ---
 
-You want to benchmark the storage of your Ceph cluster(s)? This is a short list of tools to benchmark storage.
+The first point is to evaluate the hardware of interest, we evaluate the following three parameters:
 
-**Recommended tools**:
+* raw disk performance
+* network performance
+* CPU and memory utilization
+
+**RAW DISK PERFORMANCE**
+
+fio (Flexible I/O Tester) is a versatile I/O benchmark tool that can be used to
+simulate various I/O workloads on storage systems. When testing raw disks with
+fio, you typically bypass the filesystem and directly access the block device.
+Here's a basic example of using fio to test a raw disk:
+
+```
+fio --name=mytest --ioengine=sync --rw=randwrite --bs=4k --size=1G --numjobs=16 --time_based --runtime=30s
+```
+
+Let's break down the options used in this example:
+--name=mytest: A user-defined name for the job.
```

Comment:
This did not format well, making it difficult to read. You could try a definition format (using dl, dt, dd HTML tags).

<dl>
  <dt>--ioengine=sync</dt>
  <dd>Specifies the I/O engine...</dd>
</dl>